src/components/product/OutOfStock/OutOfStock.tsx

```diff
@@ -24,48 +26,94 @@ export interface OutOfStockProps {
   title?: string
   /**
    * Notification icon.
-   * @default <Icon name="Bell" />
+   * @default <Icon name="BellRinging" />
    */
   icon?: ReactElement
+  /**
+   * Notification icon.
+   * @default <Icon name="BellRinging" />
+   */
+  notificationIcon?: ReactElement
   /**
    *
    */
   onSubmit: (value: string) => void
 }
 
 function OutOfStock(props: OutOfStockProps) {
+  const defaultButtonTxt = 'Notify me'
```

Comment:
I don't think we need to abbreviate `text` here, so far we have been using the full name. What do you think?

```suggestion
  const defaultButtonText = 'Notify me'
```

Reply:
Makes total sense ✨